File: services/user_management/firebase_config.js
import {initializeApp} from 'firebase/app';
import {getFirestore} from 'firebase/firestore';
import {getAuth} from 'firebase/auth';
// Your web app's Firebase configuration
// For Firebase JS SDK v7.20.0 and later, measurementId is optional
const firebaseConfig = {
  apiKey: '',
  authDomain: '',
  projectId: '',
  storageBucket: '',
  messagingSenderId: '',
  appId: '',
  measurementId: '',
};

// Initialize Firebase
const app = initializeApp(firebaseConfig);
const db = getFirestore(app);
const auth = getAuth(app);

export {db, auth};

File: services/task_management/app/config/firebase_config.py
import os
import firebase_admin
from firebase_admin import credentials, firestore

# Firebase local configuration
credentials_path = os.getenv('CREDENTIALS_PATH')
cred = credentials.Certificate(credentials_path)
app = firebase_admin.initialize_app(credential=cred)
db = firestore.client(app=app)

# Initialize Firestore client
# app = firebase_admin.initialize_app()
# db = firestore.client()